qa/integration-tests/src/test/java/io/camunda/zeebe/it/exporter/ExporterSerializationTest.java

```diff
@@ -11,66 +11,45 @@
 
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import io.camunda.zeebe.broker.exporter.debug.DebugHttpExporter;
-import io.camunda.zeebe.broker.system.configuration.BrokerCfg;
-import io.camunda.zeebe.broker.system.configuration.ExporterCfg;
 import io.camunda.zeebe.it.clustering.ClusteringRule;
 import io.camunda.zeebe.protocol.jackson.ZeebeProtocolModule;
 import io.camunda.zeebe.protocol.record.Record;
 import io.camunda.zeebe.test.util.WorkloadGenerator;
 import io.camunda.zeebe.test.util.record.RecordingExporter;
-import io.camunda.zeebe.test.util.socket.SocketUtil;
 import java.io.IOException;
-import java.net.URL;
-import java.util.Collections;
 import java.util.List;
-import java.util.Map;
 import java.util.stream.Collectors;
-import org.awaitility.Awaitility;
+import org.apache.logging.log4j.util.Strings;
 import org.junit.Rule;
 import org.junit.Test;
 
 public final class ExporterSerializationTest {
   private static final ObjectMapper MAPPER =
       new ObjectMapper().registerModule(new ZeebeProtocolModule());
-  @Rule public ClusteringRule clusteringRule = new ClusteringRule(1, 1, 1, this::configureBroker);
-  private int debugExporterPort;
+  @Rule public ClusteringRule clusteringRule = new ClusteringRule(1, 1, 1);
 
   @Test
   public void shouldDeserializeExportedRecords() throws IOException {
-    // when
-    WorkloadGenerator.performSampleWorkload(clusteringRule.getClient());
-
-    // then
-    // collecting all exported records will wait up to 2 seconds before returning, giving us some
-    // assumption for now that everything has been exported
+    // given
     RecordingExporter.setMaximumWaitTime(2_000);
+    WorkloadGenerator.performSampleWorkload(clusteringRule.getClient());
     final var exportedRecords = RecordingExporter.records().collect(Collectors.toList());
-    final var exportedCount = exportedRecords.size();
-    final var deserializedRecords = fetchJsonRecords(exportedCount);
-    assertThat(deserializedRecords).hasSameSizeAs(exportedRecords);
+    final var jsonRecords = exportedRecords.stream().map(Record::toJson).toList();
+    final var jsonString = "[" + Strings.join(jsonRecords, ',') + "]";
```

Comment:
```suggestion
    final var jsonString = "[" + String.join(jsonRecords, ',') + "]";
```

I assume you meant to use the built-in `String` and not the log4j util `Strings` ;)

Reply:
Lol :laughing: yes thanks for the hint :+1: Switching languages is always fun :D